tests/test_uploads.py

```diff
@@ -116,6 +117,7 @@ def files(self):
             self.assertEqual(rv.status_code, status_code)
 
     def testBase64ScreenshotUploads(self):
+        '''Test should check screenshot is Base64.'''
```

Comment:
This isn't quite right as well. 

`self.assertEqual(rv.status_code, status_code)` is checking that we get the expected return status for some bad base64 images, and a good one.